denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -123,11 +123,15 @@ def is_selecting_zapper_func(self, event):
 
     @log_fsm_condition
     def is_selecting_bind_zapper(self, event):
-        return '我要綁定補蚊燈！' == event.message.text
+        return '我要綁定捕蚊燈！' == event.message.text
```

Comment:
Just to make sure. Can these zapper related condition functions be triggered by pure text instead of template button?

Reply:
Sry, i don't get it.
Do you mean you want to know if template button can trigger these functions?